freqtrade/freqtradebot.py

```diff
@@ -831,7 +831,7 @@ def _notify_enter(self, trade: Trade, order: Dict, order_type: Optional[str] = N
             'type': msg_type,
             'buy_tag': trade.enter_tag,
             'enter_tag': trade.enter_tag,
-            'exchange': self.exchange.name.capitalize(),
+            'exchange': trade.exchange.capitalize(),
```

Comment:
i honestly don't see what benefit this brings. Freqtrade won't support multiple exchanges (and we don't aim to - within one process).

So please describe why you think this is better than using the current approach - otherwise i don't see why this would be better than the prior approach.